contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2548,15 +2600,41 @@ WHERE
   ps.dbid = sys.db_id();
 GRANT SELECT ON sys.partition_schemes TO PUBLIC;
 
+CREATE OR REPLACE VIEW sys.destination_data_spaces as
+SELECT
+  ps.scheme_id as partition_scheme_id,
+  cast(s.n as int) as destination_id,
+  cast(1 as int) as data_space_id -- primary filegroup
+FROM 
+  sys.babelfish_partition_scheme ps
+INNER JOIN 
+  sys.partition_functions pf ON pf.name = ps.partition_function_name
+CROSS JOIN 
+  generate_series(1, pf.fanout + cast(ps.next_used as int)) s(n)
```

Comment:
Need more explanation for this

Reply:
discussed it offline, it is need for DDL export to generate partition scheme with correct filegroups.